experimental/basisLieHighestWeight/MB.jl

```diff
@@ -0,0 +1,364 @@
+module MB
+export basisLieHighestWeight2
+export is_fundamental
+
+using Polymake
```

Comment:
What is this needed for?

Reply:
The user is only supposed to use the two exported functions, the rest is part of the algorithm.